docs/develop/pivottables.md

```diff
@@ -179,6 +179,48 @@ The following code snippet adds a slicer for the "Type" field. It sets the selec
 
 :::image type="content" source="../images/slicer.png" alt-text="A slicer filtering data on a PivotTable.":::
 
+### Value field settings for summaries
+
+Change how the PivotTable summarizes and displays data with these settings. The field in each data hierarchy can display the data in different ways, such as percentages, standard deviations, and relative comparisons.
+
+#### Summarize by
+
+The default summarization of a data hierarchy field is as a sum. `DataPivotHierarchy.setSummarizeBy` lets you combine the data for each row or column in a different way. [`AggregationFunction`](/javascript/api/office-scripts/excelscript/excelscript.aggregationfunction) lists the all the available options.
+
+The following code snippet changes "Crates Sold Wholesale" to show each item's standard deviation, instead of the sum.
+
+```typescript
+  const wholesaleSales = farmPivot.getDataHierarchy("Sum of Crates Sold Wholesale");
+  wholesaleSales.setSummarizeBy(ExcelScript.AggregationFunction.standardDeviation);
+```
+
+#### Show values as
+
+`DataPivotHierarchy.setShowAs` applies a calculation to the values of a data hierarchy. Instead of the default sum, you can show values or percentages relative to other parts of the PivotTable.
+
+The following code snippet changes the display for "Crates Sold at Farm". The values will be shown as a percentage of the grand total for the field.
+
+```typescript
+  const farmSales = farmPivot.getDataHierarchy("Sum of Crates Sold at Farm");
+  farmSales.setShowAs({
+    calculation: ExcelScript.ShowAsCalculation.percentOfGrandTotal
+  });
+```
+
+Some `ShowAsRule`s need another field or item in that field as a comparison. The following code snippet again changes the display for "Crates Sold at Farm". This time, the field will show each value's difference from the value of the "Lemons" in that farm row. If a farm has not sold any lemons, the field shows "#N/A".
```

Comment:
I find the appearance of ShowAsRule here confusing. With a bit of research I've determined that setShowAs takes a ShowAsRule as a parameter. Could we make this connection more clear in this article section? I think adding two things would suffice: 
- a link to DataPivotHierarchy.setShowAs 
- a sentence that says setShowAs takes ShowAsRule as a parameter, and the first example uses the `calculation` property of ShowAsRule

Reply:
Great feedback! I changed the example to be more explicit about setting that type. I also added a sentence and link about the `ShowAsRule` obejct.

Reply:
Looks good!